pkgs/development/tools/unity3d/base.nix

```diff
@@ -0,0 +1,150 @@
+# Args
+{ source
+, ver, build
+, extraLibs ? [ ]
+, preloadLibs ? [ ]
+}:
+
+{ stdenv, lib, makeWrapper, file, getopt
+, gtk2, gtk3, gdk_pixbuf, glib, libGL, libGLU, nss, nspr, udev, tbb
+, alsaLib, GConf, cups, libcap, fontconfig, freetype, pango
+, cairo, dbus, expat, zlib, libpng12, nodejs, gnutar, gcc, gcc_32bit
+, libX11, libXcursor, libXdamage, libXfixes, libXrender, libXi
+, libXcomposite, libXext, libXrandr, libXtst, libSM, libICE, libxcb, chromium, libpqxx
+
+}:
+
+let
+  libs64 = [gcc.cc gtk2 gdk_pixbuf glib libGL libGLU nss nspr
+    alsaLib GConf cups libcap fontconfig freetype pango
+    cairo dbus expat zlib libpng12 udev tbb
+    libX11 libXcursor libXdamage libXfixes libXrender libXi
+    libXcomposite libXext libXrandr libXtst libSM libICE libxcb
+    libpqxx gtk3
+  ] ++ extraLibs;
+  libPath64 = lib.makeLibraryPath libs64;
+  libPath32 = lib.makeLibraryPath [ gcc_32bit.cc ];
+  binPath = lib.makeBinPath [ nodejs gnutar ];
+  preloadLibsString = lib.optionalString (lib.length preloadLibs > 0) (":" + builtins.concatStringsSep ":" preloadLibs);
+
+in stdenv.mkDerivation rec {
+  name = "unity-editor-${version}";
+  version = "${ver}x${build}";
+  
+  nosuidLib = ./unity-nosuid.c;
+
+  nativeBuildInputs = [ makeWrapper file getopt ];
+
+  src = source;
+
+  outputs = [ "out" ];
+
+  sourceRoot = ".";
```

Comment:
I don't think it's needed, right?
```suggestion
```

Reply:
Oh, I see you just took the old file. But I guess it can't hurt if you make it a bit nicer :)

Reply:
removing `sourceRoot = ".";` breaks the build.
`outputs = [ "out" ];` is not needed though